src/test/java/com/employeesService/EmployeesService/EmployeesServiceApplicationTests.java

```diff
@@ -1,13 +1,37 @@
 package com.employeesService.EmployeesService;
 
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 
+import javax.persistence.EntityManager;
+
 @SpringBootTest
 class EmployeesServiceApplicationTests {
 
-	@Test
-	void contextLoads() {
-	}
+    private EntityManager entityManager;
+
+    @Autowired
+    public EmployeesServiceApplicationTests(EntityManager entityManager) {
+        this.entityManager = entityManager;
+    }
+
+    @Test
+    void contextLoads() {
+    }
+
+//    @Test
+//    public Optional<Department> testDepartmentThrowsEmptyResultDataAccessException(String deptNo) {
+//        Throwable e = null;
+//
+//        try {
+//            return Optional.ofNullable(this.entityManager.find(Department.class, deptNo));
+//        } catch (Throwable ex) {
+//            e=ex;
+//            return Optional.empty();
+//        }
+//
+//        assertTrue(e instanceof EmptyResultDataAccessException);
+//    }
```

Comment:
Very good that you've tried to verify something with Tests 👍 

Did it work? Why did you leave it commented?

Reply:
^^